Userland/Shell/Tests/valid.sh

```diff
@@ -78,7 +78,7 @@ pushd sh-test
     if not test "$(echo x(a*))" = "xad xae xaf"  { fail globs in lists do not resolve to lists }
     if not test "$(echo "foo"a*)" = "fooad fooae fooaf"  { fail globs join to dquoted strings }
 popd
-rm -fr sh-test
+rm -fr /tmp/sh-test
```

Comment:
Reminds me that the shell doesn't have the equivalent of `atexit`/`trap`...#7339.